mkat_fpga_tests/test_cbf.py

```diff
@@ -91,11 +111,37 @@ def test_channelisation(self):
         # Channel magnitude responses for each frequency
         chan_responses = []
         last_source_freq = None
+        QDR_error_roaches = set()
+
+        def get_pfb_counts(status_dict):
+            pfb_list = []
```

Comment:
Why not just make a dict and fill in the values in the loop? Would make the code easier to read too.